engine/gfx/cgb_layouts.asm

```diff
@@ -271,6 +271,10 @@ _CGB_FinishBattleScreenLayout:
 	ld a, PAL_BATTLE_BG_ENEMY_HP
 	call FillBoxWithByte
 
+	; set nuzlocke icon palette
+	hlcoord 0, 1, wAttrmap
+	ld [hl], PAL_BATTLE_BG_EXP_GENDER
+
```

Comment:
This should be one `ldcoord_a 0, 1, wAttrmap` line below, along with the other three.